src/export/general/generalExport.go

```diff
@@ -174,7 +175,19 @@ func (data *OverallStats) updateData() error {
 // ExportJSON exports data to a JSON file for a specified number of iterations
 // and a specified refreshed rate.
 func ExportJSON(filename string, iter uint32, refreshRate uint64) error {
-	os.Remove(filename)
+	fmt.Println("Here", filename)
+	if _, err := os.Stat(filename); err == nil {
+		fmt.Printf("Previous profile with name 'grofer_profile' exists. Overwrite? (Y/N) ")
```

Comment:
The filename can be something that the used might have used earlier as well, need not be `grofer_profile` right?

Reply:
Ah yes, I'll fix that